lendingbot.py

```diff
@@ -283,6 +326,42 @@ def createLoanOffer(cur,amt,rate):
 loanOrdersRequestLimit = {}
 defaultLoanOrdersRequestLimit = 200
 
+def amountToLent(activeCurTestBalance,activeCur,lendingBalance,lowrate):
+	restrictLent = False
+	if(activeCur in coincfg and coincfg[activeCur]['maxtolentrate'] == 0 and lowrate > 0 or lowrate <= coincfg[activeCur]['maxtolentrate'] and lowrate > 0):
```

Comment:
This line causes errors if the user is not using coincfg. I was able to solve them by using parenthesis to clarify the and/or logic.
`	if((activeCur in coincfg) and ((coincfg[activeCur]['maxtolentrate'] == 0 and lowrate > 0) or (lowrate <= coincfg[activeCur]['maxtolentrate'] and lowrate > 0))):`